src/backends/ilp-quoter/index.js

```diff
@@ -30,9 +30,13 @@ class ILPQuoter {
 
   * putPair (uri) {
     const req = request({ method: 'PUT', uri, json: true })
-    const res = yield req
-    if (res.statusCode >= 400) {
-      throw new UnsupportedPairError(req.uri)
+    try {
+      const res = yield req
+      if (res.statusCode >= 400) {
```

Comment:
I think what you want is something like this:

```
const res = yield req
if (res.statusCode >= 400) {
  log.error('Unsupported Pair: ' + uri + ' ' + err.message)
  throw new UnsupportedPairError(req.uri)
}
```

Otherwise, the error will not bubble up to the request handler